force-app/main/default/flows/After_Insert_Update_Event_Registration.flow-meta.xml

```diff
@@ -3,8 +3,8 @@
     <actionCalls>
         <name>Event_Reg_Notification</name>
```

Comment:
@t-gul I could not see any decision element on the flow which checks the "canceled" checkbox. It seems whenever an event registration is updated, the relevant session registrations will be canceled by the flow automatically. Is that correct?

Reply:
@sgoEma I will check it. If there is a problem like that, I will update flow

Reply:
@sgoEma Flow is updated.